app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] You have a view for confirmation at [app/views/devise/confirmations/new.html.erb](https://github.com/Nahid1911/blog-app-ruby-on-rails/pull/10/files#diff-eb9d2217df62505f3e24b1eff9e71afc7559219ebd549d7fbcb902bb245f2582) , but you need to set the module `confirmable` in `user.rb`

Reply:
Resolved